lib/dal/src/func/binding.rs

```diff
@@ -366,3 +415,11 @@ impl WsEvent {
         WsEvent::new(ctx, WsPayload::LogLine(payload)).await
     }
 }
+
+// TODO(nick): centralize this.
+async fn publish_immediately(ctx: &DalContext, ws_event: WsEvent) -> WsEventResult<()> {
+    let subject = format!("si.workspace_pk.{}.event", ws_event.workspace_pk());
+    let msg_bytes = serde_json::to_vec(&ws_event)?;
+    ctx.nats_conn().publish(subject, msg_bytes.into()).await?;
+    Ok(())
+}
```

Comment:
We should never be doing this. Just a port of the original code. Will need to cull in a second pass.